### PR TITLE
WASI-NN get output and compute

### PR DESCRIPTION
**Pull request by Ahmedounet**
Includes all the wasi nn functions

Includes task SP-224

### CODE REVIEW

File: core/iwasm/libraries/wasi-nn/wasi_nn_tensorflow.cpp [hunk 1]
```diff
@@ -38,8 +38,43 @@ _load(graph_builder_array builder, graph_encoding encoding)
...
+
+	interpreter->Invoke();
+
+    return success;
+}
+
+
+uint32_t _get_output(graph_execution_context context, uint32_t index, tensor tensor){
+
+    auto* output = interpreter->typed_output_tensor<float>(0);
```
Comment (tonibofarull) [inline core/iwasm/libraries/wasi-nn/wasi_nn_tensorflow.cpp:75]: where is this output going?

**Comment by tonibofarull** [comment]
Please, rebase!

**Review by tonibofarull — APPROVED** [review]

**Review by tonibofarull — CHANGES_REQUESTED** [review]
rebase

File: core/iwasm/libraries/wasi-nn/wasi_nn_tensorflow.cpp (hunk 1 above)
Comment (Ahmedounet) [reply core/iwasm/libraries/wasi-nn/wasi_nn_tensorflow.cpp:75]: Made the change, we pass a pointer and modify it's value

File: core/iwasm/libraries/wasi-nn/wasi_nn.c [hunk 2]
```diff
@@ -72,13 +72,45 @@ wasi_nn_set_input(
...
+
+uint32_t
+wasi_nn_compute(wasm_exec_env_t exec_env , graph_execution_context context)
+{
+
+    wasm_module_inst_t instance = wasm_runtime_get_module_inst(exec_env);
+
+    graph_execution_context context = wasm_runtime_addr_app_to_native(instance, context);
+
+    return _compute(_compute(context));
```
Comment (tonibofarull) [inline core/iwasm/libraries/wasi-nn/wasi_nn.c]: why 2 computes?

**Review by tonibofarull — COMMENTED** [review]
ok!

**Pull request merged**